terra_layer/style.py

```diff
@@ -455,31 +554,58 @@ def generate_style_from_wizard(layer, config):
                 'With "graduated" symbology, "boundaries" or "method" should be provided'
             )
 
+        include_no_value = config.get("include_no_value", True)
         if boundaries:
+            field_style = get_field_style(field)
+            no_value_fill_color = config.get(
+                "no_value_fill_color", DEFAULT_NO_VALUE_FILL_COLOR
+            )
+
             style = gen_layer_fill(
-                fill_color=gen_style_steps(get_field_style(field), boundaries, colors),
+                field_style,
+                fill_color=gen_style_steps(field_style, boundaries, colors),
                 fill_opacity=config.get("fill_opacity", DEFAULT_FILL_OPACITY),
                 stroke_color=config.get("stroke_color", DEFAULT_STROKE_COLOR),
+                include_no_value=include_no_value,
+                no_value_fill_color=no_value_fill_color,
+                no_value_fill_opacity=config.get(
+                    "no_value_fill_opacity", DEFAULT_NO_VALUE_FILL_OPACITY
+                ),
+                no_value_stroke_color=config.get(
+                    "no_value_stroke_color", DEFAULT_NO_VALUE_STROKE_COLOR
+                ),
             )
 
             legend_addition = {
-                "items": gen_legend_steps(boundaries, colors)[::-1],
+                "items": gen_legend_steps(
+                    boundaries, colors, include_no_value, no_value_fill_color
+                )[::-1],
             }
             return (style, legend_addition)
         else:
-            return (DEFAULT_STYLE_GRADUADED, {})
+            return (
+                DEFAULT_STYLE_GRADUADED,
+                {"items": [DEFAULT_LEGENDE_GRADUADED]} if include_no_value else {},
+            )
 
     elif symbology == "circle":
         """ config = {
             "field": "my_field",
             "symbology": "circle",
+            "include_no_value": False,  # Show no value features on map and legend
             "max_diameter": 200,
             "fill_color": "#0000cc",
             "fill_opacity": 0.5,
             "stroke_color": "#ffffff",
             "stroke_width": 1,
+            "no_value_fill_color": "#000000",
```

Comment:
can you use a dict like:

```python
{
  "no_value_style" : {
    "fill_color": …
  }
}
```

Reply:
Keep it flat to use with default values.

Reply:
Don't see the point...